playwright/test/exhibition-guides.test.ts

```diff
@@ -12,8 +12,8 @@ const bslCookie: CookieProps[] = [
   },
 ];
 
-const newJasonGuideId = 'ZthrZRIAACQALvCC';
-const newJasonGuideRelativeURL = `/guides/exhibitions/${newJasonGuideId}`;
+const newJasonGuideUid = 'jason-and-the-adventure-of-254';
```

Comment:
Think this fall within this conversation https://github.com/wellcomecollection/wellcomecollection.org/issues/11241, so easy to make it fail isn't it?

Reply:
Also very optional, but I should've probably removed the "new" prefix when I deleted the old guide 🤦‍♀️ , we could do that here?

Reply:
Some of the tests would fail if the uid was changed yes, I don't think it's a likely scenario though.
Otherwise I guess we'd have to fetch the exhibition with its id, find out what it's uid is from the response and then write the tests to use that. Do you want me to do that in this PR?

Reply:
Yeah as discussed, let's go with hard-coded UIDs for now and see if they become an issue